pkg/influxunifi/integration_test_expectations.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
Since these files are only edited by humans, I'd suggest any language besides json. yaml is probably the most forgiving. Or add a `main` test package that generates these files? mmmm